### PR TITLE
Implement DDPG

### PR DESCRIPTION
**Pull request by kengz**
- [x] allow environments with continuous actions in Lab; fix dims
- [x] add `actor, target_actor, critic, target_critic` models in DDPG
- [x] fix dimensions, get pendulum running without DDPG updates
- [x] update `critic` according to DDPG algo with the `K mean` loss function and `y` computation
- [x] update `actor` using sampled gradient
- [x] update target networks by weight adjustment parametrized by `tau`
- [x] refactor policy out as `noise` policy

**Delegate to next PR**
- [ ] biject to do discrete actions too
- [ ] do batch normalization
- [ ] see if we can make it theano-runnable too

non-DDPG stuff:
- [x] fix CI by copying keras.json
- [x] stop saving `loss` in `sys_vars` to keep files lean; will blow up when running Atari
- [x] rename `hidden_layers` to unify param name
- [x] rename `size_first_hidden_layer`
- [x] refactor agents building, fix missing optimizer clone for doubledqn
- [x] add order category for plots, esp architecture
- [x] plan the next experiments: `dqn, rand_dqn, rand_dqn_v1, rand_sarsa, acrobot , rand_mountain_dqn, rand_lunar_dqn`

**Delegate to next PR**
- [ ] fix ideal_fitness_score

### CODE REVIEW

**Comment by kengz** [comment]
@lgraesser 2 more `update` steps in the DDPG algorithm to implement (see checklist above); pretty close to a working one.
